Shared/Account/AccountLoginView.swift

```diff
@@ -52,6 +52,10 @@ struct AccountLoginView: View {
                     .padding()
             } else {
                 Button(action: {
+                    let secureProtocolPrefix = "https://"
+                    if !server.hasPrefix(secureProtocolPrefix) {
```

Comment:
I think we should check for `http://` too, and only prepend something if neither `http` nor `https` is given. I don't imagine many people will be running insecure instances, but I could imagine situations like testing on a local network, where you're accessing an instance at `http://10.0.0.10`.

Reply:
Yes! Good catch — I'm making changes now.

Reply:
@thebaer Given that [insecure connections are blocked by default](https://github.com/writeas/writefreely-swiftui-multiplatform/issues/71#issuecomment-698927571), any attempt to login over http will fail. I propose that for the 1.0 release, if an http connection is attempted, we should try to upgrade to https (and put up an alert if it fails). What do you think?

Reply:
👇 Added those changes but will leave this conversation for you to resolve.

Reply:
Okay, no problem. The current solution looks good for now!